db/migrate/20230503003534_add_user_to_comment.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :user, null: false, foreign_key: true
+  end
```

Comment:
- [ ] As requested by the previous reviewer, The associations between models must be configured correctly according to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Since we are using the `author_id` column to reference the modal user, we need to show that the `'author_id'` references the User class and update the value of the 'foreign_key' attribute to `{ to_table: :users }`. You can write the query like this: 


```ruby
   add_reference :comments, :author, null: false, foreign_key: { to_table: :users }
```